agreement/actions.go

```diff
@@ -113,6 +113,9 @@ func (a networkAction) String() string {
 	if a.t() == ignore || a.t() == disconnect {
 		return fmt.Sprintf("%v: %5v", a.t().String(), a.Err)
 	}
+	if a.Tag == protocol.ProposalPayloadTag {
+		return fmt.Sprintf("%v: %2v: %5v", a.t().String(), a.Tag, a.CompoundMessage.Proposal.value())
```

Comment:
nit: use %s for strings and %v for objects.

Reply:
done! There's a lot of cases in `actions.go` where this nit would apply, but let's leave that for another PR.